README.md

```diff
@@ -69,38 +77,36 @@ The following is an example setup using the POA Sokol testnet as the Home networ
-  * Generate a new Ethereum keystore file JSON and its password.
-    * Got to https://www.myetherwallet.com.
-    * Enter a password and click `Create New Wallet`.
-    * Download the keystore file.
-
-3. Create a new account in Nifty Wallet or MetaMask and import the JSON keystore file. See the wallet resources if you need more information on this step.  
-
-4. Fund the test account(s).
-  * Fund Home accounts (`Validators`) using the [POA Sokol Faucet](https://faucet-sokol.herokuapp.com/)
-  * Get free Kovan Coins from the [gitter channel](https://gitter.im/kovan-testnet/faucet) or [Iracus faucet](https://github.com/kovan-testnet/faucet) for Foreign Accounts. Get 5 Keth to 1 acc, and transfer from it to all other wallets.
-
-
-5. Deploy the Sokol <-> Kovan Bridge contracts.
-  * Go to the the `sokol-kovan-bridge` folder created in step 1 and `git clone https://github.com/poanetwork/poa-bridge-contracts`
-  * Follow instructions in the [POA Bridge contracts repo](https://github.com/poanetwork/poa-bridge-contracts).
-  * Set the parameters in the .env file.
-    * DEPLOYMENT_ACCOUNT_PRIVATE_KEY: Private key from step 2
-    * HOME_RPC_URL=https://sokol.poa.network
-    * HOME_OWNER_MULTISIG: Wallet address(es) of validators. For testing, you can use the same address for all address values in the file. 
-    * FOREIGN_RPC_URL=https://kovan.infura.io/mew
-
-
-5. Install and run the POA Bridge Oracle.
-  * Got to the `sokol-kovan-bridge` folder and  `git clone https://github.com/poanetwork/token-bridge`
-  * Follow instructions in the [POA NodeJS Oracle repo](https://github.com/poanetwork/token-bridge).
-
-
-If successful, you will see bridge processes run when you issue a command. For example run `npm run watcher:signature-request`
+2. Prepare temporary ETH address(es) for deployment by creating new account(s) in Nifty Wallet or MetaMask. See the [wallet resources](#resources) if you need more information on this step.  
+
+3. Fund the test account(s).
+    * Fund Home accounts (`Validators`) using the [POA Sokol Faucet](https://faucet-sokol.herokuapp.com/)
+    * Get free Kovan Coins from the [gitter channel](https://gitter.im/kovan-testnet/faucet) or [Iracus faucet](https://github.com/kovan-testnet/faucet) for Foreign Accounts. Get 5 Keth to 1 acc, and transfer from it to all other wallets.
+
+4. Deploy the Sokol <-> Kovan Bridge contracts.
+    * Go to the the `sokol-kovan-bridge` folder created in step 1 and `git clone https://github.com/poanetwork/poa-bridge-contracts`
+    * Follow instructions in the [POA Bridge contracts repo](https://github.com/poanetwork/poa-bridge-contracts).
+    * Set the parameters in the .env file.
+      * `DEPLOYMENT_ACCOUNT_PRIVATE_KEY`: Export the private key from step 2
+      * `HOME_RPC_URL`=https://sokol.poa.network
+      * Wallet address(es) for the validators. For testing, you can use the same address for all address values in the file. This includes:
```

Comment:
These are not addresses for the validators. They are for bridge contracts management wallet.